src/components/layers/toolbar/__tests__/LayerToolbar.spec.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { LayerToolbar } from '../LayerToolbar';
+
+describe('LayerToolbar', () => {
+    it('Should render only a visibility toggle and opacity slider', () => {
+        const wrapper = shallow(
```

Comment:
You could save some lines of code by wrapping the shallow in a function. Here is an example:
https://github.com/dhis2/data-visualizer-app/blob/master/packages/app/src/components/Visualization/__tests__/BlankCanvas.spec.js